04/cipher_caesar.py

```diff
@@ -0,0 +1,41 @@
+diacritics = ['á', 'é', 'í', 'ó', 'ú', 'ý', 'č', 'ď', 'ě', 'ň', 'ř', 'š', 'ť', 'ž', 'ů']
+ASCII_numbers = [49, 50, 51, 52, 53, 54, 55, 56, 57, 58]
+
+key = input('key ')
```

Comment:
U šifry je práve na kontrolu vstupu dobré použiť while cyklus, ktorý beží dovtedy kým sa nezadá vstup správne. Overenie či je vstup celé číslo môžeš skúsiť pomocou try except bloku ktorý si už používala v iných úlohách.

Reply:
Najjednoduchšie je to vysvetlené tu:

Reply:
Samozrejme ide to vyriešiť aj bez try except bloku, nie je to jediná cesta :slightly_smiling_face: